en/docs/deploy/self-hosted/openshift.md

```
@@ -0,0 +1,213 @@
+```bash
+export DOCKER_DEFAULT_PLATFORM=linux/amd64
+```
+
+This setting applies only to the current terminal session.
+:::
+
+## How Code to Cloud works
+
+When you build with `cloud = "openshift"`, the compiler generates OpenShift manifests alongside the executable JAR:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)